gabbi/tests/__init__.py

```diff
@@ -10,7 +10,3 @@
 # WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 # License for the specific language governing permissions and limitations
 # under the License.
-
-import six
-
-six.add_move(six.MovedModule('mock', 'mock', 'unittest.mock'))
```

Comment:
This change causes another pep8 failure. Easiest fix is to remove all the content in the file, leaving it empty.

Reply:
done